packages/klighd-core/src/depth-map.ts

```diff
@@ -172,7 +172,18 @@ export class DepthMap {
 
         if (element.parent === element.root && element instanceof KNode) {
             const providedRegion = new Region(element)
-            providedRegion.absolutePosition = element.bounds
+
+            let topdownScaleFactor = 1
+            if ((element.parent as any).properties == undefined || (element.parent as any).properties['org.eclipse.elk.topdown.scaleFactor'] == undefined) {
+                topdownScaleFactor = 1
+            } else {
+                topdownScaleFactor = (element.parent as any).properties['org.eclipse.elk.topdown.scaleFactor']
+            }
```

Comment:
the scale factor already is 1, so just inverting the condition would make an else clause unnecessary. Or, why not write this as a one-liner with a fallback like this: `const topdownScaleFactor = (element.parent as any).?properties['org.eclipse.elk.topdown.scaleFactor'] ?? 1`

And: is this `any` cast necessary?

(multiple occurances of this in this PR)

Reply:
good suggestion for simplifying the code,  I'll have to check the any cast